src/test/java/com/inqoo/trainingservice/app/service/SubcategoryServiceIT.java

```diff
@@ -17,93 +22,137 @@
 import java.util.UUID;
 
 import static org.assertj.core.api.Assertions.assertThat;
+
 @SpringBootTest
 @Transactional
 class SubcategoryServiceIT {
     @Autowired
     private SubcategoryService subcategoryService;
     @Autowired
     private CategoryService categoryService;
+    @Autowired
+    private CategoryConverter categoryConverter;
```

Comment:
Try to avoind using converters as Spring beans. Try to use them by yourself or change to static calls

Reply:
done